src/main.rs

```diff
@@ -376,6 +376,18 @@ fn run() -> Result<(), failure::Error> {
                         .short("e")
                         .long("env")
                         .takes_value(true)
+                )
+                .arg(
+                    Arg::with_name("no_worker")
```

Comment:
these could be the same

Reply:
nit: these names tend to be kebab case: `no-worker`

Reply:
i think i misunderstood things, and was thinking this was the variable name I'd get. Obvs that is ridiculous and this is just for display.